server/transport.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2014 The Cockroach Authors.
```

Comment:
It seems like this might be better named raft_transport.go or perhaps just raft.go.

Also, change copyright to 2015.